manifests/init.pp

```diff
@@ -129,6 +130,9 @@
   }
 
   # validate params
+  validate_integer($vas_conf_vasypd_update_interval)
+  $vas_conf_vasypd_update_interval_integer = 0 + $vas_conf_vasypd_update_interval # str2integer
```

Comment:
Unsure if this 'trick' will work in the future.
Usually I recommend to use stdlib's floor() instead [1]. See ssh module for an example [2]

[1] https://github.com/puppetlabs/puppetlabs-stdlib#floor
[2] https://github.com/ghoneycutt/puppet-module-ssh/blob/v3.49.1/manifests/init.pp#L472